src/test/java/io/knotx/stack/KnotxServerTester.java

```diff
@@ -94,7 +95,9 @@ public void testGet(VertxTestContext context, Vertx vertx, String url, Map<Strin
     MultiMap headersMultiMap = MultiMap.caseInsensitiveMultiMap();
     headersMultiMap.addAll(headers);
 
-    WebClient client = WebClient.create(vertx);
+    WebClientOptions clientOptions = new WebClientOptions();
```

Comment:
Instead of hardcoding this option here, we can do:
```
void testGet(VertxTestContext context, Vertx vertx, String url, Consumer<HttpResponse<Buffer>> assertions) {  }
void testGet(VertxTestContext context, WebClient webClient, String url, Consumer<HttpResponse<Buffer>> assertions) {  }
```

Reply:
I refactored the whole class. Please take a look now.